CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Master
 
+##### New Rule
```

Comment:
Please add new rules to the 'Enhancements' section down below instead of creating a new section.